src/Microsoft.ML.Recommender/MatrixFactorizationTrainer.cs

```diff
@@ -302,24 +304,29 @@ internal static class Defaults
         /// <summary>
         /// The <see cref="TrainerInfo"/> contains general parameters for this trainer.
         /// </summary>
-        public override TrainerInfo Info { get; }
+        TrainerInfo ITrainer.Info => _info;
```

Comment:
> TrainerInfo ITrainer.Info => _info; [](start = 7, length = 36)

Here and in FAFM we hide TrainerInfo.
In same time TrainerEstimatorBase class keep having it.
I would prefer to be consistent and make TrainerInfo public. Unless you have reason not to do that.

Reply:
Hmm okay fair enough.

Reply:
Let's keep it explicit for now. The fact that it's not producing an `ISingleFeaturePredictionTransformer` complicates the story a bit, and I'd rather not force a solution prior to v1.

Reply:
So, to be clear, despite initially looking like a mistake I think it's correct and intentional.